brightcove-exoplayer/360VideoSampleApp/src/main/AndroidManifest.xml

```diff
@@ -10,7 +10,8 @@
         android:allowBackup="true"
         android:icon="@drawable/ic_launcher"
         android:label="@string/app_name"
-        android:theme="@style/AppTheme" >
+        android:theme="@style/AppTheme"
+        android:usesCleartextTraffic="true">
```

Comment:
Would it be more useful to document (somewhere) that the AndroidManifest can be modified to use this attribute if HTTP traffic is allowed, rather than apply this change globally?

Reply:
@BCjwhisenant  That's documented here: https://support.brightcove.com/android-pie-support-native-sdk-android
Honestly, I added it to all sample apps because it was easier and faster than to test all individual sample apps to check if the captions and poster images were using https. I'm not so worry about the video source, since we added default selection of https and I've seen https on all VideoCloud responses I've debugged.
If you think this is important, I'd like to ask Kinnari's help with this task.